src/shared/ui/select/Select.tsx

```
@@ -0,0 +1,106 @@
+          xmlns="http://www.w3.org/2000/svg"
+          className={twMerge(
+            'lucide lucide-chevron-down h-4 w-4 opacity-50 transition-transform duration-200',
+            dropdown && 'rotate-180'
+          )}
+          fill="none"
+          viewBox="0 0 24 24"
+          stroke="currentColor"
+          strokeWidth="2"
+          strokeLinecap="round"
+          strokeLinejoin="round"
+        >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)